### PR TITLE
client: fix invalid node name pattern regex

### PR DESCRIPTION
**Pull request by andresilva**
See https://gitlab.parity.io/parity/mirrors/polkadot/-/jobs/3206771.

polkadot companion: https://github.com/paritytech/polkadot/pull/7484

(the companion above is fake, needed to unlock ci)

### CODE REVIEW

File: client/cli/src/commands/run_cmd.rs [hunk 1]
```diff
@@ -404,7 +404,7 @@ pub fn is_node_name_valid(_name: &str) -> std::result::Result<(), &str> {
 		return Err("Node name should not contain invalid chars such as '.' and '@'")
 	}
 
-	let invalid_patterns = r"^https?:\/\/";
+	let invalid_patterns = r"^https?:/";
```
Comment (ggwpez) [inline client/cli/src/commands/run_cmd.rs:407]:
```suggestion
	let invalid_patterns = r"^https?:";
```

Anything starting with `http:` or `https:` is already bad, or?  
Btw we do not check whether the name is non-empty 🙈

**Review by ggwpez — APPROVED** [review]
Thanks! We will have to merge it red.

**Comment by michalkucharczyk** [comment]
Are other protocols fine (like 'wss://')?

**Review by bkchr — APPROVED** [review]

**Comment by bkchr** [comment]
> Are other protocols' prefixes fine (like 'wss://')?

In the end this entire check here is useless anyway, because everyone could send whatever they want to the telemetry backend. So, this here is just some kind of sanity check.

**Comment by bkchr** [comment]
bot merge force

**Pull request merged**